src/features/ajax/aggregate/index.js

```diff
@@ -119,11 +106,19 @@ export class Aggregate extends AggregateBase {
         event.spanTimestamp = xhrContext.dt.timestamp
       }
 
-      // if the ajax happened inside an interaction, hold it until the interaction finishes
-      if (this.spaNode) {
-        var interactionId = this.spaNode.interaction.id
-        spaAjaxEvents[interactionId] = spaAjaxEvents[interactionId] || []
-        spaAjaxEvents[interactionId].push(event)
+      const spaFeature = getFeatureState({ agentIdentifier, featureName: FEATURE_NAMES.basicSpa })
+      const {
+        interactions
+      } = spaFeature?.hasInteraction?.({ timestamp: event.startTime }) || {}
+
+      // if the ajax happened inside an ixn window (found a match), add it to the ixn
+      if (interactions?.length) {
```

Comment:
What if there is more than 1 interaction that's returned? How can it make sense that there's more than 1 interaction at a time and that this ajax is a child of every one (multiple)?

Reply:
it could be a child of both a page load interaction and a click interaction if both are happening simultaneously while the ajax occurs under this model, since its timestamp-based alone.